testing/describes.js

```diff
@@ -556,7 +571,13 @@ class RealWinFixture {
         interceptEventListeners(win.document.documentElement);
         interceptEventListeners(win.document.body);
         env.interceptEventListeners = interceptEventListeners;
-
+        if (!(spec.xhrMock === false)) {
+          fetchMock.constructor.global = env.win;
+          fetchMock._mock();
+          env.expectFetch = function(url, response) {
+            return fetchMock.mock(url, response);
+          };
+        }
```

Comment:
Let's move this block of code to a top-level helper function to avoid code dupe.